keras/layers/advanced_activations.py

```diff
@@ -338,7 +338,11 @@ def call(self, inputs, mask=None):
 
       # Since we are adding it to the raw scores before the softmax, this is
       # effectively the same as removing these entirely.
-      inputs += adder
+      try:
```

Comment:
Rather than try/except, do an explicit shape check. Note that if the shape is dynamic, no error can be raised until actual runtime (e.g. the try/except would not work).

When raising an error message:

- Print what the input shapes were (give context about what happened)
- Print what they should have been (state how to fix the issue)
- Make sure to add a unit test that checks the message is raised as expected (`AssertRaises` with a regex)